### PR TITLE
Tweak use_lifecycle()

### LINKED ISSUE #1012: use_lifecycle has a confusing instruction
When I run,

```r
use_lifecycle()
```
One of the instructions is:

```
Refer to functions with `lifecycle::fun()`
```

What does this mean? Functions in my package? So instead of `\link{foo}`, I should write `\link{experimental::foo}`??

### PR DESCRIPTION
**Pull request by hadley**
* Don't display default package usage hints
* Don't normalize description

Fixes #1012

### CODE REVIEW

File: R/helpers.R [hunk 1]
```diff
@@ -117,8 +117,7 @@ use_rd_macros <- function(package) {
 
   if (!package %in% macros) {
     macros <- c(macros, package)
-    macros <- paste0("    ", macros, collapse = ",\n")
-    desc::desc_set(RdMacros = macros, file = proj, normalize = TRUE)
+    desc::desc_set(RdMacros = macros, file = proj)
```
Comment (lionel-) [inline R/helpers.R]: I see

```
   [ OK: 438 | SKIPPED: 21 | WARNINGS: 1 | FAILED: 1 ]

  1. Failure: use_lifecycle() respects existing RdMacros field 
```

Why this change?

Comment (hadley) [reply R/helpers.R]: Ooops, I missed that test — but setting `normalize = TRUE` normalises the entire `DESCRIPTION` which feels a bit aggressive
